docs/tables/aws_application_signals_service_level_objective.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Steampipe Table: aws_application_signals_service_level_objective - Query AWS Application Signals using SQL"
+description: "Allows users to query AWS Application Signals to retrieve detailed information about each SLO, including its name, ARN, attainment_goal, goal and sli."
+---
+
+# Table: aws_application_signals_service_level_objective - Query AWS Application Signals Service Level Objective using SQL
+
+## Table Usage Guide
+
+The `aws_application_signals_service_level_objective` table in Steampipe provides you with information about SLOs within AWS Application Signals. This table allows you to query SLO details, including the name, ARN, attainment_goal, goal and sli, and associated metadata.
+
+## Examples
+
+### Basic info
+
+```sql+postgres
+select
+  arn,
+  name,
+  attainment_goal,
+  goal,
+from
+  aws_application_signals_service_level_objective
+```
```

Comment:
Could you please include the query in both PostgreSQL and SQLite format?

Reply:
Sure thing.  Is there a way to check my documentation is correct? or just by inspection?